src/main/cli/cli.c

```diff
@@ -2703,6 +2707,58 @@ static void cliVtx(const char *cmdName, char *cmdline)
 
 #endif // VTX_CONTROL
 
+#ifdef USE_VTX_SMARTAUDIO
+static void cliVtxOpModel(const char *cmdName, char *cmdline)
+{
+    if (isEmpty(cmdline)) {
+        // If you change the model this value will be stale until reboot.
+        cliPrintLine(saDevice.willBootIntoPitMode ? "race" : "free");
+    } else {
+        bool freeModel = strncasecmp(cmdline, "free", 4) == 0;
+        bool pir = strncasecmp(cmdline, "pir", 3) == 0 || strncasecmp(cmdline, "race", 4) == 0;
```

Comment:
I think describing this in the manuals will be complicated anyway. In SA 2.1 (at least as it's modeled in the existing `cms_menu_vtx_smartaudio.c` code) there's really just `free` and `race`. In the older SA 2.0 you had to choose between two race modes, i.e. `pir` and `por`. With 2.1, `por` has gone away and what was `pir` is now essentially what you get with `race`.

Many VTXs out there are version 2.0 and for these it's best to think in terms of `free`, `pir` and `por`. For newer 2.1 VTXs it makes more sense to think in terms of just `free` and `race`.

If I have to choose between `pir` and `free` then I'll choose `free` as that better matches the newer VTXs that will presumably eventually dominate. And anyway, once set, you can't currently distinguish between `pir` and `por` (as you can see above the existing `willBootIntoPitMode` value is just a boolean that distinguishes between `free` and not `free`).

For the documentation, I'd write something like:

> Use `vtx_op_model` without an argument to retrieve the current pit mode:
>
> ```
> # vtx_op_model
> free
> ```
>
> Use `vtx_op_model` with an argument to change whether or not the VTX will start in pit mode:
>
> ```
> # vtx_op_model race
> ```
>
> For SmartAudio 2.1 VTXs, you can choose between the pit modes `free` and `race`. For older 2.0 VTXs there's an additional pit mode called `por` (out-range pit mode), if this mode is selected then the VTX will use the frequency specified by `vtx_pit_mode_freq` when it starts in pit mode. If you specify `race` then the pit mode is in-range and the value of `vtx_pit_mode_freq` is ignored. See the exiting Betaflight SmartAudio [documentation](https://github.com/betaflight/betaflight/wiki/Unify-Smartaudio) for more details.

Reply:
**Update:** I claimed above that:

> once set, you can't currently distinguish between `pir` and `por` (as you can see above the existing `willBootIntoPitMode` value is just a boolean that distinguishes between free and not free).

This was incorrect - I've changed my `willBootIntoPitMode` check above to also check `smartAudioDevice_s.mode` so it can distinguish in-range and out.